src/store/plugins/ui/observables.js

```diff
@@ -87,6 +87,22 @@ export default (store) => {
       refCountDelay(1000),
     );
 
+  const middlewareStatus$ = sdk$
+    .pipe(
+      switchMap(sdk => timer(0, 30000).pipe(map(() => sdk))),
+      switchMap(async sdk => (sdk ? (await sdk.middleware.getMdwStatus()).OK : true)),
+      multicast(new BehaviorSubject(true)),
+      refCountDelay(1000),
+    );
+
+  const middlewareQueueLength$ = sdk$
+    .pipe(
+      switchMap(sdk => timer(0, 30000).pipe(map(() => sdk))),
+      switchMap(async sdk => (sdk ? (await sdk.middleware.getMdwStatus()).queueLength : 0)),
+      multicast(new BehaviorSubject(0)),
+      refCountDelay(1000),
+    );
```

Comment:
Let's combine these observables to don't make double requests (of sdk.middleware.getMdwStatus) in cases when we subscribed to both of them.